addons/account/tests/test_account_all_l10n.py

```diff
@@ -47,6 +47,6 @@ def test_all_l10n(env):
         _logger.info('Testing COA: %s (company: %s)', template_code, company.name)
         try:
             with env.cr.savepoint():
-                env['account.chart.template'].try_loading(template_code, company, install_demo=True)
```

Comment:
You need to install the dependencies of all these modules first in order to have the fields existing when installing the templates

Reply:
nevermind, dependences are installed by design :facepalm: